modules/play-json/src/main/scala/com/evolutiongaming/skafka/producer/JsonProducer.scala

```diff
@@ -1,23 +1,18 @@
 package com.evolutiongaming.skafka.producer
 
+import cats.Applicative
 import com.evolutiongaming.skafka.{ToBytes, Topic}
 import play.api.libs.json.{JsValue, Json}
 
-import scala.concurrent.Future
-
 trait JsonProducer[F[_]] {
   def apply(record: ProducerRecord[String, JsValue]): F[RecordMetadata]
 }
 
 object JsonProducer {
 
-  val Empty: JsonProducer[Future] = apply(Producer.Send.Empty)
+  def empty[F[_] : Applicative]: JsonProducer[F] = apply(Producer.Send.Empty[F])
```

Comment:
Send.Empty -> Send.empty

Reply:
Did this replacement globally, as it has already been inconsistent.